docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_IOS.mdx

```diff
@@ -87,52 +87,47 @@ The Frame Processor Plugin will be exposed to JS through the `VisionCameraProxy`
 
 ![Xcode "Create Bridging Header" alert](https://docs-assets.developer.apple.com/published/7ebca7212c/2a065d1a-7e53-4907-a889-b7fa4f2206c9.png)
 
-3. Inside the newly created Bridging Header, add the following code:
-
-```objc
-#import <VisionCamera/FrameProcessorPlugin.h>
-#import <VisionCamera/Frame.h>
-```
-
-4. In the Swift file, add the following code:
+3. In the Swift file, add the following code:
 
 ```swift
+import VisionCamera
```

Comment:
Here I found that there's no need to use bridging header. Instead we can use VisionCamera as a Swift module.

For plugin implemented inside an application codebase nothing more is needed.

For plugin implemented as a library, VisionCamera needs to be referenced as a podspec dependency (I've already [mentioned it as one of post-steps](https://github.com/mateusz1913/vision-camera-plugin-builder/pull/39/files#diff-6eefc925eb2820a960acc6aff9d4f54af5cc66e446ebb6a480daac62f89ab8baR291) in the PR to vision-camera-plugin-builder)